frontend/src/components/Header.vue

```diff
@@ -3,7 +3,11 @@
     <v-app-bar class="elevation-0">
       <v-toolbar-title>{{headerTitle}}</v-toolbar-title>
       <v-spacer></v-spacer>
-      <v-btn v-if="isGeneric" text x-small color="primary" class="ma-2" @click="openExploreGenerators">Explore and Install Generators...</v-btn>
+      <v-btn v-if="isGeneric" text x-small color="primary" @click="openExploreGenerators">
```

Comment:
what is the behavior if removing the text prop ?
<v-btn v-if="isGeneric" x-small color="primary" @click="openExploreGenerators">

Reply:
It shows like this:
![image](https://user-images.githubusercontent.com/35297731/88782225-ffca4600-d195-11ea-8166-8998d365fec9.png)